src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,3 +1,10 @@
-export const UserInfo = () => (
-  <>Put the user here</>
+import './UserInfo.scss';
+
+export const UserInfo = ({ post }) => (
```

Comment:
The component UserInfo is dependent on the entire post object, but it only uses post.user.name. Consider changing the component to take in only the necessary data.

Reply:
Same here
![2023-08-31_15h18_57](https://github.com/mate-academy/react_static-list-of-posts-js/assets/61904995/a5ad3100-6830-4120-ad0b-855325fe9b42)